sycl/include/sycl/detail/pi.h

```diff
@@ -83,9 +83,9 @@
 // 12.25 Added PI_EXT_DEVICE_INFO_ATOMIC_FENCE_ORDER_CAPABILITIES and
 // PI_EXT_DEVICE_INFO_ATOMIC_FENCE_SCOPE_CAPABILITIES for piDeviceGetInfo.
 // 12.26 Added piextEnqueueReadHostPipe and piextEnqueueWriteHostPipe functions.
-// 12.27 Added new queue create and get APIs for immediate commandlists
-// piextQueueCreate2, piextQueueCreateWithNativeHandle2,
-// piextQueueGetNativeHandle2
+// 12.27 Added properties parameter to piextQueueCreateWithNativeHandle and
+// changed native handle type of piextQueueCreateWithNativeHandle and
+// piextQueueGetNativeHandle
```

Comment:
Please make it a new breaking change instead, i.e. 13.31

Reply:
OK, done.